src/puppetlabs/trapperkeeper/services/status/status_proxy_service.clj

```diff
@@ -0,0 +1,16 @@
+(ns puppetlabs.trapperkeeper.services.status.status-proxy-service
+  (:require [clojure.tools.logging :as log]
+    [puppetlabs.trapperkeeper.core :refer [defservice]]))
+
+(defservice status-proxy-service
+  [[:WebroutingService add-proxy-route]
+   [:ConfigService get-in-config]]
+  (init [this context]
+    (add-proxy-route
+      this
+      {:host (get-in-config [:status-proxy :target-host])
```

Comment:
Rather than just exposing this in external configuration as essentially a serialized form of the arguments that the `add-proxy-route` function takes, I think it might be better to model the external configuration on an existing service that has proxy support.  For example, this could look like the "proxy-config" blob that the `reverse-proxy-ca-service` uses - https://github.com/puppetlabs/pe-puppet-server-extensions/blob/master/documentation/config_settings.markdown#caconf.  By doing it this way, we might be able to more easily migrate to a different underlying representation of the proxy mechanism later / abstract away some detailed configuration options in `add-proxy-route` that the `status-proxy-service` wouldn't need to use.